juju/loop.py

```diff
@@ -6,7 +6,7 @@
 import warnings
 
 warnings.warn(
-    "juju.loop module is being deprecated by 3.0, use juju.jasyncio instead",
+    "juju.loop module is being deprecated by 3.0, use juju._jasyncio instead",
```

Comment:
this file can be removed instead

Reply:
Should we just say "use asyncio instead"?

Reply:
True, but I'm going to remove `juju/loop.py` entirely in a separate PR.